fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -261,11 +261,12 @@ protected Response exceptionResponse(FHIROperationException e, Status status) {
             status = issueListToStatus(e.getIssues());
         }
 
+        // Only log full stack trace if server (5xx) error
         if (status.getFamily() == Status.Family.SERVER_ERROR) {
             log.log(Level.SEVERE, e.getMessage(), e);
         } else {
             if (log.isLoggable(Level.INFO)) {
-                log.log(Level.INFO, e.getMessage(), e);
+                log.log(Level.INFO, e.getMessage());
             }
```

Comment:
are we sure this is the answer? 

it would be better to split the logging... if Level.fine log the stacktrace

Reply:
Sure, I can have it do that instead.

Reply:
That'd be great